packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/edit-name/edit-name.html

```diff
@@ -21,6 +21,9 @@
                 name="databaseName"
                 class="oui-input"
                 data-ng-model="$ctrl.name"
+                data-minlength="$ctrl.minNameLength"
```

Comment:
**question:** Is there any reason to combine both `minlength` and `ng-maxlength`? Why not prefer using `ng-minlength`?

Reply:
Actually we have to use ng-minlength, good catch ! Changed here: https://github.com/ovh/manager/pull/5971/commits/b071cfc248e286ec0589cda41c911c489817e935